loader.js

```diff
@@ -115,21 +115,23 @@ define([
 
     // install optional plugins, summary widget is handled separately as it was added long ago
     if (config.plugins) {
-      if (
-        config.plugins.summaryWidgets === true ||
-        config.plugins.summaryWidgets?.enabled === true
-      ) {
+      if (config.plugins.summaryWidget === true || config.plugins.summaryWidget?.enabled === true) {
         openmct.install(openmct.plugins.SummaryWidget());
       }
 
       Object.entries(config.plugins).forEach(([plugin, pluginConfig]) => {
-        const pluginExists = openmct.plugins[plugin] || openmct.plugins.example[plugin];
+        const examplePluginExists = openmct.plugins.example[plugin];
+        const pluginExists = openmct.plugins[plugin] || examplePluginExists;
         const pluginEnabled = pluginConfig?.enabled;
-        const isSummaryWidget = plugin === 'summaryWidgets';
+        const isSummaryWidget = plugin === 'summaryWidget';
```

Comment:
Why do we need to treat `SummaryWidget` as special anymore?

Reply:
You know... I was thinking for backwards compatibility, but this is a whole new config all around, so I guess we can just do it however we want, so I'll remove the special handling.